File: user/src/main/java/com/moonlyte/myjavalibrary/model/Posts.java
package com.moonlyte.myjavalibrary.model;

import com.google.gson.annotations.SerializedName;

import java.util.ArrayList;
import java.util.List;

public class Posts {
    public List<Comments>commentsList=new ArrayList<>();
    @SerializedName("userId")
    public Integer userId;
    @SerializedName("id")
    public Integer id;
    @SerializedName("title")
    public String title;
    @SerializedName("body")
    public String body;
    private boolean expanded;
    public void setExpanded(boolean expanded) {
        this.expanded = expanded;
    }
    public boolean isExpanded() {
        return expanded;
    }
}

File: app/src/main/java/com/moonlyte/socialapp/features/post/model/Posts.kt
package com.moonlyte.socialapp.features.post.model

import androidx.room.ColumnInfo
import androidx.room.Entity
import androidx.room.PrimaryKey
import com.google.gson.annotations.SerializedName

@Entity(tableName = "posts")
data class Posts(
    @ColumnInfo(name = "userId") @SerializedName("userId") val userId: Int,
    @PrimaryKey(autoGenerate = false)  @SerializedName("id") val id: Int,
    @ColumnInfo(name = "title") @SerializedName("title") val title: String,
    @ColumnInfo(name = "body") @SerializedName("body") val body: String
) {


}
